Yoast/wordpress-seo
# PC-351 Fix automated tests failing because of missing Ryte class

## Context


*

## Summary


This PR can be summarized in the following changelog entry:

* Fixes an unreleased bug where the plugin couldn't be activated because of a fatal error regarding the deprecated Ryte integration.

## Relevant technical choices:

*

## Test instructions

### Test instructions for the acceptance test before the PR gets merged
This PR can be acceptance tested by following these steps:

* See the integrations test pass.
* create a zip with `grunt artifact`
* install and activate it on e.g. a Local by Flywheel instance
  * see you don't get a fatal error and the plugin is correctly activated.

#### Relevant test scenarios


### Test instructions for QA when the code is in the RC




QA can test this PR by following these steps:

* install and activate the RC containing this fix on e.g. a Local by Flywheel instance
  * see you don't get a fatal error and the plugin is correctly activated.

## Impact check

This PR affects the following parts of the plugin, which may require extra testing:

*

## UI changes


## Other environments


## Documentation


## Quality assurance


Fixes [PC-351]


[PC-351]: https://yoast.atlassian.net/browse/PC-351?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
